spec/services/invoices/calculate_fees_service_spec.rb

```diff
@@ -814,7 +837,9 @@
           aggregate_failures do
             expect(result).to be_success
 
-            expect(invoice.fees.charge_kind.count).to eq(0)
+            # Why not? Is this test correct?
+            # pp subscription.created_at.to_date, subscription.started_at.to_date, subscription.subscription_at.to_date, timestamp.to_date
```

Comment:
Layout/LineLength: Line is too long. [138/120] (https://rubystyle.guide#max-line-length)